src/modules/infobox/components/styling/DrawingStyleColorSelector.vue

```diff
@@ -1,33 +1,39 @@
 <template>
-    <div
-        class="color-select-box bg-light border-light"
-        :class="{ inline: inline }"
-        data-cy="drawing-style-color-select-box"
-    >
-        <div
+    <div class="color-select-box bg-light border-light" data-cy="drawing-style-color-select-box">
+        <ButtonWithIcon
             v-for="color in colors"
             :key="color.name"
             :data-cy="`color-selector-${color.name}`"
+            :button-font-awesome-icon="['fas', 'circle']"
+            :style="{ color: color.name }"
+            :primary="currentColor === color"
```

Comment:
You should probably implement a better check than "currentColor === color" (maybe implement an equals function that checks the values instead of the reference) as for now if you close the drawing, reload the page, and reopen the drawing, the selection is lost (no button is highlighted anymore).

Reply:
Very good catch

Reply:
I corrected it thanks @jedef